src/templates/ui/index.ts

```diff
@@ -13,15 +13,69 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export const defaultHtml = `<!DOCTYPE html>
+export const defaultHtml = (appName: string): string => `<!DOCTYPE html>
 <html>
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Actyx-Pond-Demo!</title>
+    <title>${appName}</title>
+    <style>
+      .app {
```

Comment:
app is not used

Reply:
Thanks for noticing it! Now converted in a wrapper class so we can keep the spinning wheel in the center of the viewport :)